ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -394,13 +394,13 @@ public void handleDisconnect(
       final DisconnectReason reason,
       final boolean initiatedByPeer) {
     if (ethPeers.registerDisconnect(connection)) {
-      LOG.debug(
-          "Disconnect - {} - {} - {}... - {} peers left\n{}",
+      LOG.info(
```

Comment:
how frequent is this one? asking to understand the impact on log UX

Reply:
yeah this might be too much for mainnet
```
2023-11-28 10:52:39.667+10:00 | nioEventLoopGroup-3-8 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0xa7577f327e190458... - 17 peers left
2023-11-28 10:52:50.513+10:00 | nioEventLoopGroup-3-3 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x01 TCP_SUBSYSTEM_ERROR - 0xe108725c42e2aa86... - 16 peers left
2023-11-28 10:52:51.226+10:00 | vert.x-eventloop-thread-0 | WARN  | EngineQosTimer | Execution engine not called in 120 seconds, consensus client may not be connected
2023-11-28 10:52:54.222+10:00 | nioEventLoopGroup-3-10 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x7a0718e2d7d4dc19... - 16 peers left
2023-11-28 10:52:58.426+10:00 | EthScheduler-Timer-0 | INFO  | PivotSelectorFromSafeBlock | Waiting for consensus client, this may be because your consensus client is still syncing
2023-11-28 10:53:22.017+10:00 | nioEventLoopGroup-3-3 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x2bf0042844b851bb... - 16 peers left
2023-11-28 10:53:24.321+10:00 | nioEventLoopGroup-3-4 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x971eaefe449b291f... - 18 peers left
2023-11-28 10:53:24.643+10:00 | nioEventLoopGroup-3-7 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x73fef41ee12d7fab... - 18 peers left
2023-11-28 10:53:35.287+10:00 | nioEventLoopGroup-3-10 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x01 TCP_SUBSYSTEM_ERROR - 0x442bbbf796325434... - 17 peers left
2023-11-28 10:53:39.201+10:00 | nioEventLoopGroup-3-2 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x01 TCP_SUBSYSTEM_ERROR - 0x47c7f2a91ce28a6a... - 16 peers left
2023-11-28 10:53:54.364+10:00 | nioEventLoopGroup-3-10 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x971eaefe449b291f... - 17 peers left
2023-11-28 10:53:54.690+10:00 | nioEventLoopGroup-3-6 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x73fef41ee12d7fab... - 16 peers left
2023-11-28 10:53:58.522+10:00 | EthScheduler-Timer-0 | INFO  | PivotSelectorFromSafeBlock | Waiting for consensus client, this may be because your consensus client is still syncing
2023-11-28 10:54:01.195+10:00 | nioEventLoopGroup-3-1 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x42f5a99e0c676592... - 16 peers left
2023-11-28 10:54:02.486+10:00 | nioEventLoopGroup-3-1 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0xd2403e175cf04655... - 16 peers left
2023-11-28 10:54:18.531+10:00 | nioEventLoopGroup-3-1 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x7ad8d195880d5a33... - 16 peers left
2023-11-28 10:54:24.366+10:00 | nioEventLoopGroup-3-5 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x971eaefe449b291f... - 17 peers left
2023-11-28 10:54:35.919+10:00 | nioEventLoopGroup-3-4 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x01 TCP_SUBSYSTEM_ERROR - 0x6fbc31e076e0ca78... - 16 peers left
2023-11-28 10:54:37.110+10:00 | nioEventLoopGroup-3-8 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x856d25c9592a8b38... - 16 peers left
2023-11-28 10:54:37.483+10:00 | nioEventLoopGroup-3-6 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0xd8482b28dba149a1... - 16 peers left
2023-11-28 10:54:51.224+10:00 | vert.x-eventloop-thread-0 | WARN  | EngineQosTimer | Execution engine not called in 120 seconds, consensus client may not be connected
2023-11-28 10:54:54.533+10:00 | nioEventLoopGroup-3-5 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x971eaefe449b291f... - 17 peers left
2023-11-28 10:54:58.600+10:00 | EthScheduler-Timer-0 | INFO  | PivotSelectorFromSafeBlock | Waiting for consensus client, this may be because your consensus client is still syncing
2023-11-28 10:54:58.684+10:00 | nioEventLoopGroup-3-8 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x01 TCP_SUBSYSTEM_ERROR - 0x108097a033125037... - 16 peers left
2023-11-28 10:55:09.313+10:00 | nioEventLoopGroup-3-5 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x2ad62e0b344afecc... - 16 peers left
2023-11-28 10:55:25.805+10:00 | nioEventLoopGroup-3-7 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x971eaefe449b291f... - 16 peers left
2023-11-28 10:55:47.271+10:00 | nioEventLoopGroup-3-2 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x36cee0f8d9c6e883... - 16 peers left
2023-11-28 10:55:55.116+10:00 | nioEventLoopGroup-3-10 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0xaa49d5021cfa3d6a... - 16 peers left
2023-11-28 10:55:58.680+10:00 | EthScheduler-Timer-0 | INFO  | PivotSelectorFromSafeBlock | Waiting for consensus client, this may be because your consensus client is still syncing
2023-11-28 10:56:02.899+10:00 | nioEventLoopGroup-3-8 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0xcee5d198ccbed5b6... - 16 peers left
2023-11-28 10:56:11.524+10:00 | nioEventLoopGroup-3-3 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0xd938bcbb37907241... - 16 peers left
2023-11-28 10:56:24.750+10:00 | nioEventLoopGroup-3-9 | INFO  | EthProtocolManager | Disconnect - Outbound - 0x03 USELESS_PEER - 0x971eaefe449b291f... - 16 peers left
2023-11-28 10:56:51.223+10:00 | vert.x-eventloop-thread-0 | WARN  | EngineQosTimer | Execution engine not called in 120 seconds, consensus client may not be connected
```

Reply:
changed back to debug